bodhi/tests/functional/test_updates.py

```diff
@@ -1147,7 +1147,7 @@ def test_edit_update(self, publish, *args):
         self.assertEquals(up['date_approved'], None)
         self.assertEquals(up['date_pushed'], None)
         self.assertEquals(up['locked'], False)
-        self.assertEquals(up['alias'], None)
+        self.assertEquals(up['alias'], u'FEDORA-2015-0002')
```

Comment:
Do you want to use ``time.localtime`` here for the year?  I.e., will these tests break when we get to 2016?